proto/rpc.proto

```diff
@@ -25,5 +25,11 @@ message InviteResponse {
 }
 
 message DeviceInfo {
+  enum DeviceType {
```

Comment:
Comments in linked PR for why we're not implementing this. Interpretation of this value should be:

- "No value set" = device type is unknown (we have not received it yet, or the peer has not shared their device type)
- "UNRECOGNIZED" = the protobuf parser has encountered a value that it does not understand, likely because this is from a future version of the app with a deviceType that is not defined here.